docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1221,17 +1213,167 @@ paths:
             $ref: '#/definitions/Problem'
         '409':
           description: |
-            Conflict. There are no empty slots for this subscriptions. The amount of consumers for this subscription
-            already equals the maximal value - the total amount of this subscription partitions.
+            Conflict. There are several possible reasons for receiving this status code:
+
+            - There are no empty slots for this subscriptions. The amount of consumers for this subscription
+            already equals the maximum value - the total number of partitions in this subscription.
+
+            - Request to reset subscription cursors is still in progress.
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Access is forbidden for the client or event type
+          schema:
+            $ref: '#/definitions/Problem'
+
+    post:
+      tags:
+        - subscription-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        GET with body.
+
+        Starts a new stream for reading events from this subscription. The minimal consumption unit is a partition, so
+        it is possible to start as many streams as the total number of partitions in event-types of this subscription.
+        The position of the consumption is managed by Nakadi. The client is required to commit the cursors he gets in
+        a stream.
+
+        If you create a stream without specifying the partitions to read from - Nakadi will automatically assign
+        partitions to this new stream. By default Nakadi distributes partitions among clients trying to give the equal
```

Comment:
`s/the equal amount/an equal number/`

Reply:
fixed https://github.com/zalando/nakadi/pull/834/commits/db3635b998900d5554a1ba7c5f8e704fb3dff20d